cmd/explaintest/r/explain_easy.result

```diff
@@ -352,3 +352,16 @@ create table t(a bigint primary key);
 explain select * from t where a = 1 and a = 2;
 id	count	task	operator info
 TableDual_5	0.00	root	rows:0
+drop table if exists ta, tb;
+create table ta (a varchar(20));
+create table tb (a varchar(20));
+begin;
+insert tb values ('1');
+explain select * from ta where a = 1;
+id	count	task	operator info
+Projection_5	8000.00	root	test.ta.a
+└─Selection_6	8000.00	root	eq(cast(test.ta.a), 1)
+  └─UnionScan_7	10000.00	root	eq(cast(test.ta.a), 1)
```

Comment:
why not `Eval it in getSnapshotRow`?

Reply:
It can be refactored later.

Reply:
The filter `a = 1`, which is `Selection_6`, should be added as a parent of `TableReader_9`.

Reply:
This is better, we can do it in the next PR.